docs/DeveloperGuide.md

```diff
@@ -126,17 +125,6 @@ The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
```

Comment:
Referring to line 112 to the `<div markdown...` could you update the `Person` to `Book`?
For some reason GitHub prevents me from choosing the line to comment.